multibranch/pom.xml

```diff
@@ -88,7 +88,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.2.1</version>
+            <version>5.8-SNAPSHOT</version>
```

Comment:
I need `5.8-SNAPSHOT` because:

`WorkflowMultiBranchProject.DescriptorImpl` extends of `MultiBranchProjectDescriptor`, and this one extends of `AbstractFolderDescriptor`. For this relation, a priori, `WorkflowMultiBranchProject` should inherit the type of `ItemCategory` provides by `getItemCategoryId()`.

The downstream PR in this PR, [provides](https://github.com/jenkinsci/cloudbees-folder-plugin/pull/56/files#diff-b4544a5a8a71e3317534defa00e9cd95L43) this definition.